Functions.Templates/Templates/HttpTrigger-Python/README.md

```diff
@@ -0,0 +1,54 @@
+# Python HttpTrigger
+
+The Azure Functions Python HTTP Trigger creates an endpoint: `<function-host>/api/<function-name>`. When an HTTP request is made against that endpoint, the `main()` function in `__init__.py` is called.
```

Comment:
```suggestion
The HTTP trigger lets you invoke a function with an HTTP request. You can use an HTTP trigger to build serverless APIs and respond to webhooks. It creates an endpoint like  `https://<app-name>.azurewebsites.net/api/api/<function-name>`. When an HTTP request is made against that endpoint, the `main()` function in `__init__.py` is called.
```

Reply:
Accepting your suggestion by changing my own code. Just wanted to keep the commit tree clean.